components/molecule/checkboxField/src/index.scss

```diff
@@ -1,12 +1,53 @@
 @import '~@schibstedspain/sui-theme/lib/index';
+@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/spacing';
+@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/color';
+@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/box-style';
 @import '~@s-ui/react-molecule-field/lib/index';
 @import '~@s-ui/react-atom-checkbox/lib/index';
 
+$bgc-atom-checkbox-styled: $c-white !default;
+$bgc-checked-atom-checkbox-styled: $c-primary !default;
+$c-atom-checkbox-styled: str-slice(#{$c-white}, 2);
+$bgi-atom-checkbox-styled: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill="%23#{$c-atom-checkbox-styled}" d="M9 16.2L4.8 12l-1.4 1.4L9 19 21 7l-1.4-1.4L9 16.2z"></path></svg>') !default;
```

Comment:
I'm not sured about this. We should work in order to avoid this kind of things and let users put the icons using props instead.

Reply:
For the moment, I think nobody needs to change the icon. With this solution, each vertical will have the checkbox icon with his primary colour. If someone needs to change the icon, can iterate this component.

Reply:
The same thing could be achieved using a normal svg. :P

Reply:
Tell me more. How can I do that? I need to use a `background-image` on the `:before` label selector.